js/node_shared/unix.js

```diff
@@ -265,6 +265,12 @@ function unix_fork(vunit) {
   return;
 }
 
+//Provides: unix_putenv
+function unix_putenv(key, val) {
+  //TODO: nodejs equivalent?
```

Comment:
https://stackoverflow.com/questions/10829433/how-can-i-set-an-environment-variable-in-node-js